### DISCUSSION TITLE
add video player

### DISCUSSION THREAD

**Discussion by nonameboy007** [advplyr/audiobookshelf-app#812] [Ideas]
Hello everybody,

first, thank you for your tremendous work.

Do you plan to add a video player for a certain type? Indeed, I have some files, which are mp4 or avi learning videos. it would be cool to have that in audiobookshelf. do you plan to add this functionality?

Thank you for your comeback,

Regards,

**Comment by gbakeman** [comment]
My personal opinion is that this would be beyond the scope of this project, especially when there are other projects well suited for handling video libraries, such as Jellyfin.

**Comment by advplyr** [comment]
It's not planned but maybe sometime in the future. Some podcasts are video podcasts which could be another use-case. There is a lot more still to be done with the existing features and the mobile apps need to come out of beta.

Ebooks have been an ongoing "side" feature for a while where it isn't a priority but every so often small incremental improvements are made. It could end up as something like that.

**Reply by nonameboy007** [reply]
@gbakeman  and @advplyr , thanks for your feedbacks. 

So I will regulary check the changeling if I see something. Or I will try to clone the repo to edit the player. 

Thanks

**Reply by gbakeman** [reply]
Sorry @nonameboy007, kind of sounded like I was shooting down your idea and certainly didn't mean to. Hope ABS ends up being the solution you're looking for.
